### PR TITLE
Feature/suic 222/select popover

### PR DESCRIPTION
**Pull request by danivm**
[SCMI-35721](https://jira.scmspain.com/browse/SCMI-35721)

selectpopoverdemo

### CODE REVIEW

**Review by emiliovillu — APPROVED** [review]

File: components/molecule/selectPopover/src/index.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+  onAccept,
+  onCancel,
+  selectText
+}) {
+  const [isOpen, setIsOpen] = useState(false)
+
+  const popoverRef = useRef()
+
+  const selectClassName = cx(`${BASE_CLASS}-select`, {
+    [`is-open`]: isOpen,
```
Comment (midudev) [inline components/molecule/selectPopover/src/index.js]: It's weird that linter is not complaining about using template string when is not needed.

File: components/molecule/selectPopover/src/index.js [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+MoleculeSelectPopover.displayName = 'MoleculeSelectPopover'
+MoleculeSelectPopover.propTypes = {
+  acceptButtonText: PropTypes.string.isRequired,
+  cancelButtonText: PropTypes.string.isRequired,
+  children: PropTypes.node.isRequired,
+  iconArrowDown: PropTypes.node.isRequired,
+  isSelected: PropTypes.bool,
+  onAccept: PropTypes.func.isRequired,
+  onCancel: PropTypes.func.isRequired,
```
Comment (midudev) [inline components/molecule/selectPopover/src/index.js]: Why is this required? May be we can offer this to be optional, I guess, and do nothing when that happens besides closing the popover.

**Review by midudev — APPROVED** [review]

File: components/molecule/selectPopover/README.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+$ npm install @s-ui/react-molecule-select-popover --save
+```
+
+## Usage
+
+### Basic usage
+```js
+import MoleculeSelectPopover from '@s-ui/react-molecule-select-popover'
+
+return (<MoleculeSelectPopover />)
```
Comment (AitorRodriguez990) [inline components/molecule/selectPopover/README.md]: I think it doesn't work.

File: components/molecule/selectPopover/README.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+```
+
+
+> **Find full description and more examples in the [demo page](#).**
```
Comment (AitorRodriguez990) [inline components/molecule/selectPopover/README.md]: `https://sui-components.now.sh/workbench/molecule/selectPopover/demo`

**Review by AitorRodriguez990 — APPROVED** [review]

**Review by giodelabarrera — APPROVED** [review]
